scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -523,148 +523,190 @@ global.base_stats = { //tempory stats subject to change by anyone that wishes to
-			strength:20,
-			constitution:26,
-			weapon_skill : [20,5],
-			ballistic_skill : [20,5],				
-			dexterity:34,
-			intelligence:7,
-			wisdom:2,
-			charisma :2,
-			religion : "cult_mechanicus",
-			piety : 10,
-			technology :8,
-			luck :5,
-			weapon_skill : 35,
-			ballistic_skill : 40,			
-			skills: {weapons:{"ranger_long_rifle":1,}},	
-			start_gear:{"armour":"skitarii_armour", "wep1":"hellgun", "wep2":"shuriken_pistol"},
-			base_group : "skitarii",
+	"skitarii_ranger":{ // TODO rename this
+			title : "Skitarii Ranger", // TODO - that should be Eldar
+			strength : [25,1],
+			constitution : [30,2],
+			weapon_skill : [45,4],
+			ballistic_skill : [45,4],				
+			dexterity : [50,5], // Dexterity should be eldar 'specialization'
+			intelligence : [35,3],
+			wisdom : [50,3],
+			charisma : [20,2], // Arrogance from cultural stuff, supposedly
+			religion : "cult_mechanicus", // TODO - add eldar faith
+			piety : [30,5], // I think eldar rangers can be either - faithful to the path or more cynical
+			luck : 10,
+			technology : [20,1], // Elves in most fiction tend to be kind of bad at technology, right?
+			skills: {
+				weapons:{
+					"Ranger Long Rifle":3, "Shuriken Pistol":3, "Eldar Power Sword":3}}, // TODO - check if these actually work
+			start_gear:{"armour":"Power Armour", "wep1":"Ranger Long Rifle", "wep2":"Eldar Power Sword"}, // TODO - add Eldar Armour
+			base_group : "skitarii", // Might want to rename this
 	},
 	"inquisition_crusader":{
-			title : "Inquisition Crusador",
-			strength:4,
-			constitution:26,
-			dexterity:4,
-			intelligence:3,
-			wisdom:2,
-			charisma :2,
+			title : "Inquisition Crusader",
+			strength : [10,1], // 10 is considered standard for a chad-like Imperial Guardsman
+			constitution : [10,1],
+			weapon_skill : [10,1],
+			ballistic_skill : [10,1],
+			dexterity : [10,1],
+			intelligence : [10,1],
+			wisdom : [12,1], // They may know a bit more than average imperial
+			charisma : [10,1],
 			religion : "imperial_cult",
-			piety : 10,
-			technology :3,
-			luck :4,
-			weapon_skill : 20,
-			ballistic_skill : 14,			
-			skills: {},	
-			start_gear:{"armour":"Power Armour", "wep1":"Power Sword", "wep2":"Storm Shield"},
+			piety : [30,5], // Supposedly, they can be radical or puritan...
+			luck : 10,
+			technology : [8,1],
+			skills : {}, // TODO consider what skills are needed for this bloke
+			start_gear:{"armour":"Power Armour", "wep1":"Power Sword", "wep2":"Storm Shield"}, // TODO - add Light variant of Power Armour
 			base_group : "human",
 	},
+	/* TODO - add psychic capabilities
+	"sanctioned_psyker":{
+			title: "Sanctioned Psyker",
+	}
+	*/
 	"sister_of_battle":{
 			title : "Sister of Battle",
-			strength:5,
-			constitution:28,
-			dexterity:6,
-			intelligence:5,
-			wisdom:5,
-			charisma :2,
+			strength : [10,1],
+			constitution : [10,1], // TODO - consider making it that hireling armour boosts constitution, and possibly other stats
```

Comment:
And this as well.

Reply:
Yes, sororitas are glorified guard. Unless we really want to buff regular humans?